### PR TITLE
[TRANSLATION] Update Chinese translation.

### PR DESCRIPTION
**Pull request by njlyf2011**

### CODE REVIEW

File: dll/win32/ieframe/lang/zh-TW.rc [hunk 1]
```diff
@@ -29,9 +29,9 @@ IDR_BROWSE_MAIN_MENU MENU
             MENUITEM "位址列(&A)",            ID_BROWSE_BAR_ADDR
         }
     }
-    POPUP "收藏夾(&F)"
+    POPUP "我的最愛(&F)"
     {
-        MENUITEM "添加到收藏夾...(&A)",        ID_BROWSE_ADDFAV
+        MENUITEM "添加到我的最愛...(&A)",        ID_BROWSE_ADDFAV
```
Comment (katahiromz) [inline dll/win32/ieframe/lang/zh-TW.rc]: "添加到我的最愛(&A)..."

File: base/applications/rapps/lang/zh-TW.rc [hunk 2]
```diff
@@ -160,7 +160,7 @@ BEGIN
     IDS_AINFO_URLSITE "\n主頁："
```
Comment (katahiromz) [inline base/applications/rapps/lang/zh-TW.rc]: Use ASCII colons.

File: dll/win32/ieframe/lang/zh-TW.rc (hunk 1 above)
Comment (katahiromz) [reply dll/win32/ieframe/lang/zh-TW.rc]:
```suggestion
        MENUITEM "添加到我的最愛(&A)...",        ID_BROWSE_ADDFAV
```

File: base/applications/rapps/lang/zh-TW.rc (hunk 2 above)
Comment (katahiromz) [reply base/applications/rapps/lang/zh-TW.rc]:
```suggestion
    IDS_AINFO_URLSITE "\n主頁: "
```

[the remark this answers is not in the document]
Comment (katahiromz) [reply base/applications/rapps/lang/zh-TW.rc]:
```suggestion
    IDS_AINFO_URLSITE "\n主頁: "
```

**Comment by njlyf2011** [comment]
Corrected.

**Pull request merged**